README.md

```diff
@@ -1,8 +1,6 @@
 # Getting Started with the .NET MAUI Scheduler Control
 
-A quick start .NET MAUI project that shows how to create and configure the Syncfusion MAUI Scheduler to a .NET MAUI app. This project also includes a code snippet to add simple appointments to the Scheduler; change the default date, view, today’s date highlight color; and display week numbers. 
-
-Documentation: https://help.syncfusion.com/maui/scheduler/getting-started
+The [.NET MAUI Scheduler](https://www.syncfusion.com/maui-controls/maui-scheduler?utm_source=vs_marketplace&utm_medium=listing&utm_campaign=maui-scheduler-trial-vs_marketplace) has nine built-in configurable view modes that provide basic functionality for scheduling, managing, and representing appointments efficiently.A quick start .NET MAUI project that shows how to create and configure the Syncfusion MAUI Scheduler to a .NET MAUI app. This project also includes a code snippet to add simple appointments to the Scheduler; change the default date, view, today’s date highlight color; and display week numbers. 
```

Comment:
Use the bellow UTM parameter in all places
?utm_source=github&utm_medium=listing&utm_campaign=maui-scheduler-github-samples

Reply:
Updated

Reply:
Updated